vpr/src/util/vpr_utils.h

```diff
@@ -167,4 +169,6 @@ int max_pins_per_grid_tile();
 
 void pretty_print_uint(const char* prefix, size_t value, int num_digits, int scientific_precision);
 void pretty_print_float(const char* prefix, double value, int num_digits, int scientific_precision);
+
+std::vector<std::string> split_string(std::string string, char delimiter);
```

Comment:
This already exists in https://github.com/SymbiFlow/vtr-verilog-to-routing/blob/master%2Bwip/libs/libvtrutil/src/vtr_util.h#L14 ?

Reply:
Done. If everything else is fine I will merge the relative branch with this patch in master+wip and open an upstream PR.

Reply:
sounds good